### ISSUE TITLE
Zap Enchantment deals damage to the Owner and his allies (if members on the same team).

### ISSUE THREAD

**Issue by KingYamiJ** [dsfhdshdjtsb/enchantments#41]
**Describe the bug**
The enchantment loses his magic because of this "bug".

**To Reproduce**
1. step 1 Hit a zombie with a arrow and be near it.
2. step 2 The enchantment bounces on the player who shoot the arrow too, causing heavy damage.
3. step 3 If the enchantment is causing damage to the player that shoot the arrow, is very annoying to use, because you always need to be at medium-long range.

**Additional context**
Not using any other enchantment that can deal damage other than Power 1.

**Comment by memphismc** [comment]
i second this, if you are a bit too close you also get zapped, making this enchantment counteractive

**Comment by dsfhdshdjtsb** [comment]
what mod are you using for teams @KingYamiJ

**Comment by Happy1speed** [comment]
I always thought of this as a balancing feature since the enchantment is pretty good for large groups

**Comment by KingYamiJ** [comment]
> what mod are you using for teams @KingYamiJ
At that time i only used MC dungeons weapons, and some furniture mods, nothing that would conflict with bows or weapon damage type or teams. The bug occurs on every bow, even the vanilla. Was it supposed to be like this? Because is not only hurting allies, hurts you too.

**Comment by KingYamiJ** [comment]
> I always thought of this as a balancing feature since the enchantment is pretty good for large groups

Yeah, but since minecraft is more pve-sided, i don't think this is a good thing, because if an enchantment has such a disadvantage, then it would be a curse.

**Comment by Happy1speed** [comment]
> > I always thought of this as a balancing feature since the enchantment is pretty good for large groups
> 
> Yeah, but since minecraft is more pve-sided, i don't think this is a good thing, because if an enchantment has such a disadvantage, then it would be a curse.

Its still decent, you just can't let the group of enemies closer than 6 blocks.
